src/renderer/redux/actions/rewards.js

```diff
@@ -61,13 +61,6 @@ export function doClaimRewardType(rewardType) {
           reward: successReward,
         },
       });
-      if (successReward.reward_type === rewards.TYPE_NEW_USER) {
```

Comment:
Why are we getting rid of this?

Reply:
The `new_user` reward notification is handled by `rewards.js` like the others